x-pack/plugins/cloud_security_posture/public/pages/rules/use_csp_rules.ts

```diff
@@ -53,20 +53,16 @@ export const useBulkUpdateCspRules = () => {
     }: {
       savedObjectRules: RuleSavedObject[];
       packagePolicyId: CspRule['package_policy_id'];
-    }) => {
-      await savedObjects.client.bulkUpdate<RuleSavedObject>(
-        savedObjectRules.map((savedObjectRule) => ({
-          type: CSP_RULE_SAVED_OBJECT_TYPE,
-          id: savedObjectRule.id,
-          attributes: savedObjectRule.attributes,
-        }))
-      );
-      await http.post(UPDATE_RULES_CONFIG_ROUTE_PATH, {
+    }) =>
+      http.post(UPDATE_RULES_CONFIG_ROUTE_PATH, {
```

Comment:
Post mainly use for create new resource, I would consider to change it for `PUT`